tests/sles4sap/configure_selinux_policy.pm

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018-2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Configure SELinux policy according to the release notes for SLES4SAP
+#16
+# Maintainer: QE SAP <qe-sap@suse.de>
+
+use base 'selinuxtest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use version_utils qw(is_sle);
+use bootloader_setup qw(replace_grub_cmdline_settings);
+use power_action_utils;
+
+sub run {
+    my ($self) = @_;
+
+    return "SLES4SAP 15 does not support SELinux. Doing nothing." if is_sle('<16');
```

Comment:
There are `has_selinux` and `has_selinux_by_default` in `lib/version_utils` in case you want to make this more generic.

Reply:
Done!